src/openforms/scss/components/builder/_builder.scss

```diff
@@ -40,7 +40,8 @@ li.nav-item {
 }
 
 .formio-dialog,
-.ck-body .ck.ck-balloon-panel
+.ck-body .ck.ck-balloon-panel,
```

Comment:
Seems to work nicely on chrome, but on firefox I don't see the date picker :S

Reply:
mmm might be because I used the same value of another element, might have to increase that a bit. ugh, browsers...